microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -281,16 +324,21 @@ See also:
 
 ##### [.NET/C#](#tab/dotnetcsharp)
 
+* [CoreWebView2.CookieManager Property](/dotnet/api/microsoft.web.webview2.core.corewebview2.cookiemanager#microsoft-web-webview2-core-corewebview2-cookiemanager)
+
 * [CoreWebView2Cookie Class](/dotnet/api/microsoft.web.webview2.core.corewebview2cookie)
+
 * [CoreWebView2CookieList Class](/dotnet/api/microsoft.web.webview2.core.corewebview2cookielist)
```

Comment:
```suggestion
```
@novac42 @peiche-jessica - Delete link?  goes to 674 prerelease: https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2cookielist?view=webview2-dotnet-1.0.674-prerelease
This article should only list Release Stable API items.

Reply:
link deleted